### DISCUSSION TITLE
task-flink-core cpu and memory threshold settings

### DISCUSSION THREAD

**Discussion by y1q84** [cubefs/compass#212] [General]
i want to know about why the high or low cpu threshold is 0.8d and 0.4d, beside, why  the high or low memory threshold is 0.8d and 0.4d,how did you clculate it?

**Comment by y1q84** [comment]
is anyone know about it?

**Comment by nilnon** [comment]
@y1q84 This is an experience value and you can adjust it according to the actual situation.

**Reply by y1q84** [reply]
got it ,thx
